### DISCUSSION TITLE
What are the audio formats supported for wispher?

### DISCUSSION THREAD

**Discussion by omnaiduu** [openai/whisper#2292] [General]
I read a discussion where it was written anything that fmpeg support as input can be also given to whisper api? 

And we really trying to use what's app voice note, which is in opus format. Does wispher support opus?.. as fmpeg supports it and also can you update the docs

**Comment by ryanheise** [comment]
This discussion board is about the self-hosted Whisper open source software which is published in this GitHub repo, so those discussions you read (if you read them here) were probably not about the API.

The Whisper code in this repo does indeed support any input format supported by ffmpeg. As for OpenAI's API, you can search on OpenAI's own website for [details about their API.](https://platform.openai.com/docs/guides/speech-to-text)

**Comment by omnaiduu** [comment]
You were correct, i got this if I send opus file. 

`{
    "error": {
        "message": "Invalid file format. Supported formats: ['flac', 'm4a', 'mp3', 'mp4', 'mpeg', 'mpga', 'oga', 'ogg', 'wav', 'webm']",
        "type": "invalid_request_error",
        "param": null,
        "code": null
    }
}`

Can someone, add support for opus file type too. It will be great or the friction to ingrate whisper in WhatsApp will be greatly reduced and discord too.. thanks

**Reply by ryanheise** [reply]
Since you are asking about the OpenAI API which is different from what this repo is about, I suggest heading over to the OpenAI website. They have their own forums over there for their APIs. As far as the situation over "here" in this repo, there is no need to add support because this repo already supports all of the formats that ffmpeg supports.

(I've given you the link to the OpenAI website above, and you can follow this up over there.)
